lib/Command/ResetDocument.php

```diff
@@ -61,9 +60,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 		}
 
 		if ($all) {
-			$fileIds = array_map(static function (Document $document) {
-				return $document->getId();
-			}, $this->documentService->getAll());
+			$fileIds = $this->documentService->getAll();
```

Comment:
@juliushaertl I don't understand this change. `DocumentService->getAll()` returns an array of `Document` objects, not the file IDs. How is this supposed to work?

Reply:
Tested and indeed it broke with your current approach. The old approach didn't work either as `array_map()` cannot be run against a `Generator`. I pushed a fix:

```php
$fileIds = [];
$documents = $this->documentService->getAll();
foreach ($documents as $document) {
	$fileIds[] = $document->getId();
}
```